paper/paper.bib

```diff
@@ -0,0 +1,246 @@
+journal = {Journal of the American Chemical Society},
+volume = {125},
+number = {7},
+pages = {1731-1737},
+year = {2003},
+doi = {10.1021/ja026939x},
+note ={PMID: 12580598},
+URL = {https://doi.org/10.1021/ja026939x},
+eprint = {https://doi.org/10.1021/ja026939x}
+}
+
+@article{alphafold_2021,
+	title = {Highly accurate protein structure prediction with {AlphaFold}},
+	volume = {596},
+	issn = {1476-4687},
+	url = {https://doi.org/10.1038/s41586-021-03819-2},
+	doi = {10.1038/s41586-021-03819-2},
+	pages = {583--589},
+	number = {7873},
+	journaltitle = {Nature},
+	shortjournal = {Nature},
+	author = {Jumper, John and Evans, Richard and Pritzel, Alexander and Green, Tim and Figurnov, Michael and Ronneberger, Olaf and Tunyasuvunakool, Kathryn and Bates, Russ and Žídek, Augustin and Potapenko, Anna and Bridgland, Alex and Meyer, Clemens and Kohl, Simon A. A. and Ballard, Andrew J. and Cowie, Andrew and Romera-Paredes, Bernardino and Nikolov, Stanislav and Jain, Rishub and Adler, Jonas and Back, Trevor and Petersen, Stig and Reiman, David and Clancy, Ellen and Zielinski, Michal and Steinegger, Martin and Pacholska, Michalina and Berghammer, Tamas and Bodenstein, Sebastian and Silver, David and Vinyals, Oriol and Senior, Andrew W. and Kavukcuoglu, Koray and Kohli, Pushmeet and Hassabis, Demis},
+	date = {2021-08-01},
+}
+
+@article{alphafold_multi,
+author = {Richard Evans, Michael O’Neill, Alexander Pritzel, Natasha Antropova, Andrew Senior, Tim Green, Augustin Žídek, Russ Bates, Sam Blackwell, Jason Yim, Olaf Ronneberger, Sebastian Bodenstein, Michal Zielinski, Alex Bridgland, Anna Potapenko, Andrew Cowie, Kathryn Tunyasuvunakool, Rishub Jain, Ellen Clancy, Pushmeet Kohli, John Jumper, Demis Hassabis},
+title = {Protein complex prediction with AlphaFold-Multimer},
+journal = {bioRxiv},
+year = {2021},
+doi = {10.1101/2021.10.04.463034},
+URL = {https://www.biorxiv.org/content/10.1101/2021.10.04.463034v2}
+}
+
+@article{dove,
+  title={Protein Docking Model Evaluation by 3D Deep Convolutional Neural Networks},
+  author={Wang, Xiao and Terashi, Genki and Christoffer, Charles W and Zhu, Mengmeng and Kihara, Daisuke},
+  journal={Bioinformatics}
+}
+
+@article{masif,
+  title={Deciphering interaction fingerprints from protein molecular surfaces using geometric deep learning},
+  author={Gainza, P and Sverrisson, F and Monti, F and Rodol{\`a}, E and Boscaini, D and Bronstein, MM and Correia, BE},
+  journal={Nature Methods},
+  volume={17},
+  number={2},
+  pages={184--192},
+  year={2020},
+  publisher={Nature Publishing Group}
+}
+
+@article{dove_gnn,
+author={Wang, Xiao and Flannery, Sean T. and Kihara, Daisuke},    
+title={Protein Docking Model Evaluation by Graph Neural Networks},	
+journal={Frontiers in Molecular Biosciences},	
+volume={8},
+year={2021},  
+url={https://www.frontiersin.org/articles/10.3389/fmolb.2021.647915},	
+doi={10.3389/fmolb.2021.647915},	
+issn={2296-889X}
+}
+
+@article{fout_protein_nodate,
```

Comment:
This paper is also missing the publication year which causes it to show as "n.d."